deploy/charts/csi-driver/Chart.yaml

```diff
@@ -2,15 +2,28 @@ apiVersion: v2
 
 name: cert-manager-csi-driver
 type: application
-description: cert-manager-csi-driver enables issuing secretless X.509 certificates for pods using cert-manager
+description: cert-manager csi-driver enables issuing secretless X.509 certificates for pods using cert-manager
 
-home: https://github.com/cert-manager/csi-driver
+home: https://cert-manager.io/docs/usage/csi-driver
+icon: https://raw.githubusercontent.com/cert-manager/community/4d35a69437d21b76322157e6284be4cd64e6d2b7/logo/logo-small.png
+keywords:
+- cert-manager
+- csi
+- tls
+annotations:
+  artifacthub.io/alternativeName: csi-driver
+  artifacthub.io/license: Apache-2.0
+  artifacthub.io/category: security
+  artifacthub.io/recommendations: |
+    - url: https://artifacthub.io/packages/helm/cert-manager/csi-driver-spiffe
+    - url: https://artifacthub.io/packages/helm/cert-manager/cert-manager
+    - url: https://artifacthub.io/packages/helm/cert-manager/trust-manager
 maintainers:
 - name: cert-manager-maintainers
   email: cert-manager-maintainers@googlegroups.com
   url: https://cert-manager.io
 sources:
 - https://github.com/cert-manager/csi-driver
 
-appVersion: v0.6.1
-version: v0.6.1
+appVersion: v0.0.0
+version: v0.0.0
```

Comment:
How will these be set? When the chart is pushed up?

Reply:
Looks like these need to be set in the README as well (readme badge contains a version identifier)